src/main/java/se/sundsvall/notes/service/ServiceConstants.java

```diff
@@ -7,4 +7,8 @@ private ServiceConstants() {}
 	static final String REVISION_NOT_FOUND_FOR_ID_AND_VERSION = "No revision with entityId '%s' and version '%s' was found!";
 	static final String PROBLEM_DURING_DIFF = "An error occured during diff of entityId '%s' looking at version '%s' and version '%s'!";
 	static final String ERROR_NOTE_NOT_FOUND = "Note with id '%s' not found";
+
+	static final String EVENT_LOG_CREATE_NOTE = "Notering har skapats.";
```

Comment:
Notering har skapats. -> Noteringen har skapats.

?

Reply:
Made it consistent to Errand-events

Reply:
Ok, if this text is used in SupportManagement, it should probably be changed there as well. 

Either you use "bestämd form" in the text or you use "Obestämd form". It's weird to use a mix them.